.github/workflows/b2b-demo-shop-regression-api copy.yml

```
@@ -0,0 +1,69 @@
+name: "[b2b][internal] Robot Regression API DMS-ON"
+
+on:
+    schedule:
+        -   cron: '0 1 * * *'
+    workflow_dispatch:
+
+jobs:
+    docker-alpine-php-8-2-mariadb-robot-api:
+        name: "Docker / Alpine / PHP 8.2 / MariaDB / Robot / DMS API"
```

I would recommend to get rid of DMS, and use Dynamic Store || Dynamic Multistore in all those files, as well as in already existing CI files.

```suggestion
        name: "Docker / Alpine / PHP 8.2 / MariaDB / Robot / API/ Dynamic Store"
```